astropath/priors.py

```diff
@@ -118,4 +120,35 @@ def vet_theta_prior(theta_prior:dict):
     """
     chk, disallowed_keys, badtype_keys = utils.vet_data_model(
         theta_prior, theta_dmodel)
-    return chk
\ No newline at end of file
+    return chk
+
+def load_std_priors(nhalf:float=10., theta_max:float=6.):
+    """
+    Load the standard set of priors as defined
+    in the PATH paper
+
+    Args:
+        nhalf (float, optional): Number of half-light radii for analysis. Defaults to 10..
```

Comment:
"Defaults to 10.0." Just looks a little cleaner!

Reply:
edited